drivers/mil_passive_sonar/scripts/multilateration

```diff
@@ -24,30 +24,40 @@ class MultilaterationNode(object):
     def __init__(self):
         self.tfBuffer = tf2_ros.Buffer()
         self.listener = tf2_ros.TransformListener(self.tfBuffer)
-        buffer_len = rospy.get_param('~buffer_size', default=10)
+        buffer_len = rospy.get_param('~buffer_size', default=15)
+        self.minimum_delta_dist = rospy.get_param('~min_delta', default=0.2)
+        self.filter_dist = rospy.get_param('~filter_dist', default=5)
+        self.filter_count = rospy.get_param('~filter_count', default=4)
+        self.buffer_min = rospy.get_param('~buffer_min', default=10)
         self.vec_samples = deque(maxlen=buffer_len)
         self.enabled = False
-        self.global_fame = 'enu'
+        self.last_pos = None
+        self.global_frame = 'enu'
```

Comment:
As this will be a mil_common script (likely to be used on Subjugator), the global frame should also be a param (Subjugator uses "map")

Reply:
Done